functions/go/set-annotation/annotations_transformer.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2019 The Kubernetes Authors.
```

Comment:
Wrong authors for the license.

Reply:
This file is copied from kustomize repo so actually I am not clear about the license problem.

@droot @frankfarzan

Reply:
I saw we have the same issue in other transformer.go files in set-namespace and set-label directory.

Reply:
We copied these files so we need to keep these copyright lines.